src/data/messages.js

```diff
@@ -26,6 +26,51 @@ module.exports = {
         //infoPage
         'skipToSide': 'Skip to sidebar',
         'breadCrumbAria': 'You are here: ',
+        //kilometrics
+        'kilotitle': 'Find the correct rate for your kilometrics',
+        'kilolead': 'Taking your personal vehicle on a government trip? Refer to these rates.',
+        'kiloerrortitle': 'Field error or required',
+        'kiloerrortext': 'Please verify the following fields: ',
+        'provlabel': 'Province/Territory of Travel',
+        'travelledlabel': 'Kilometres Travelled',
+        'submit': 'Submit',
+        'clear': 'Clear',
+        'results1': 'The kilometric rate for ',
+        'results2': ' is ',
+        'results3': ' cents per kilometre',
+        'results4': 'For your trip of ',
+        'results5': ' kilometres you would be reimbursed ',
```

Comment:
I'm not sure if this is the right solution for this problem.  Numbered variables like that aren't a great pattern.  See if you can cook up something from the docs:  

https://formatjs.io/docs/react-intl/components/